tsconfig.json

```diff
@@ -1,17 +1,22 @@
 {
   "compilerOptions": {
-    "allowJs": true,
-    "baseUrl": ".",
+    "allowJs": false,
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "isolatedModules": true,
     "jsx": "react-jsx",
     "lib": ["DOM", "DOM.Iterable", "ES2019"],
     "moduleResolution": "node",
     "module": "ES2022",
-    "noEmit": true,
     "resolveJsonModule": true,
     "strict": true,
     "target": "ES2019"
-  }
+  },
+  "include": ["src/**/*.tsx", "src/**/*.ts", "src/**/*.jsx"],
```

Comment:
As well as compiling TypeScript we also need to compile JSX to JS as our old apps can't deal with JSX imports.